pkg/testutil/scenarios.go

```diff
@@ -19,19 +19,34 @@ import (
 type TestScenario struct {
 	API mock.API
 	Arg string
-	// will be removed once all users are using RunScenarios()
-	Args            []string
-	ConfigPath      string
-	ConfigFile      *config.File
-	DontWantOutput  string
+	// FIXME: remove once all users are using RunScenario() or RunScenarios()
+	Args []string
+	// this will be copied into global.Data.ConfigPath
+	ConfigPath string
+	// this will be copied into global.Data.ConfigFile
+	ConfigFile *config.File
+	// fail the scenario if this string appears in stdout
+	DontWantOutput string
+	// fail the scenario if any of these strings appear in stdout
 	DontWantOutputs []string
-	Name            string
-	Stdin           []string
-	WantError       string
-	WantOutput      string
-	WantOutputs     []string
+	// scenario name (appears in output when tests are executed)
+	Name string
+	// if this is supplied, NewEnv will be used to create a temporary environment
+	// for the scenario
+	NewEnv          *NewEnvConfig
 	PathContentFlag *PathContentFlag
-	SetEnv          *map[string]string
+	// set one (or more) environment variables during the scenario's execution
+	SetEnv *map[string]string
```

Comment:
The `map` type is already passed-by-pointer so you don't need to take the address of a map type when passing it around as it's already a pointer.